eo2js-runtime/src/runtime/bytes-of.js

```diff
@@ -38,19 +38,16 @@ const hexToInt = function(bytes) {
  * @return {Array.<Number>} - Adjusted byte array
  */
 const adjustNumber = function(bytes) {
-  if (bytes.length === 8) {
-    const number = bytesOf(
+  if (bytes.length !== 8) return bytes;
```

Comment:
@Akvadevka we're trying to avoid many `return` statements, [here's](https://www.yegor256.com/2015/08/18/multiple-return-statements-in-oop.html) why